devops/ansible/roles/girder/tasks/main.yml

```diff
@@ -59,6 +59,10 @@
 - include: daemon.yml
   when: girder_daemonize
 
+- name: Install Girder Plugins
+  command: "{{ girder_install_executable|default('girder-install') }} plugin {{ item.args|default('') }} {{ item.path }}"
```

Comment:
After I changed `item.path` to `item.path|default(item)`, this example usage worked for me:
```
    girder_plugins:
      - path: "{{ ansible_user_dir }}/other_plugin"
        args: "--symlink"
      - "{{ ansible_user_dir }}/myplugin"
```

Reply:
ooh, slick!  With that change, you can also preface the string with your args.

```YAML
girder_plugins:
  - "--symlink {{ ansible_user_dir }}/myplugin"
```
I'll leave it to others to decide if that's desirable or not! 😝